src/domain/storage/document/document.service.ts

```diff
@@ -188,7 +188,9 @@ export class DocumentService {
 
     const documentID = url.substring(documentsBaseUrlPath.length + 1);
     try {
-      return await this.getDocumentOrFail(documentID);
+      return await this.getDocumentOrFail(documentID, {
+        relations: { storageBucket: true },
```

Comment:
Do you really need this relation everywhere? Why not have the options as optional parameter on `getDocumentFromURL`?

Reply:
addressed